packages/mdc-slider/component.ts

```diff
@@ -22,6 +22,7 @@
  */
 
 import {MDCComponent} from '@material/base/component';
+import {applyPassive} from '@material/dom/events';
```

Comment:
You need to update @material/dom to the package.json of slider from 1.1.0 to the latest.